include/musica/micm.hpp

```diff
@@ -201,7 +198,7 @@ namespace musica
     /// @param custom_rate_parameters Array of custom rate parameters [grid cell][parameter] (various units)
     /// @param error Error struct to indicate success or failure
     void Solve(
-        auto &solver,
+        auto &solver_state_pair,
```

Comment:
Could you update the `line 192` so it can include the description for the state?

```C++
    /// @param solver Pointer to solver
```